guides/doc-Planning_Guide/topics/Org_Loc_and_Life_Cycle_Environments.adoc

```diff
@@ -1,5 +1,10 @@
+ifdef::katello[]
 [[chap-Architecture_Guide-Org_Loc_and_Life_Cycle_Environments]]
 == Organizations, Locations, and Life Cycle Environments
+endif::[]
+ifndef::katello[]
```

Comment:
the [[chap]] piece that you've omitted means you cannot link to this new non-Katello bit from elsewhere, so any references to this piece will be broken for a foreman-el build, for example.

Reply:
Right, so how should we refer to it then? For non-Katello it feels weird to have Life Cycle Environments in the URL if it isn't there in the content. Changing the chapter for Katello would break URLs.

Reply:
I added what to do below.